### PR TITLE
build: upgrade lowdb

### PR DESCRIPTION
**Pull request by alex-Arc**

### CODE REVIEW

File: apps/server/src/modules/loadDb.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ async function loadDb() {
   const dbInDisk = populateDb();
 
   const adapter = new JSONFile<DatabaseModel>(dbInDisk);
-  const db = new Low(adapter);
+  const db = new Low(adapter, dbModel);
```
Comment (alex-Arc) [inline apps/server/src/modules/loadDb.ts:56]: i'm unsure if this is correct?

File: apps/server/src/services/ConfigService.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@ class ConfigService {
   constructor() {
     this.configPath = join(getAppDataPath(), 'config.json');
     const adapter = new JSONFile<Config>(this.configPath);
-    this.config = new Low<Config>(adapter);
+    this.config = new Low<Config>(adapter, null);
```
Comment (alex-Arc) [inline apps/server/src/services/ConfigService.ts:18]: same here

Comment (cpvalente) [reply apps/server/src/services/ConfigService.ts:18]: maybe a `{}` instead?

**Review by cpvalente — COMMENTED** [review]
i think this seems correct
https://github.com/typicode/lowdb?tab=readme-ov-file#new-lowadapter-defaultdata

we could try [the preset syntax](https://github.com/typicode/lowdb?tab=readme-ov-file#typescript) if we feel is simpler. But as long as it works I dont see a need for it

**Comment by alex-Arc** [comment]
> i think this seems correct

cool, it works for me
so if you could give it a try I think we are good

**Pull request merged**